CardanoSharp.Wallet/Models/Utxo.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+using CardanoSharp.Wallet.Models.Transactions;
+
+namespace CardanoSharp.Wallet.Models
+{
+    public class Utxo
+    {
+        public string? TxHash { get; set; }
+        public uint TxIndex { get; set; }
+        public ulong Value { get; set; }
```

Comment:
Using the existing `Balance` type here (combining the Value and AssetList) will make our lives easier by having a common type for calculating values in the layers above e.g. in SingleTokenBundleStrategy.CalculateChange

Reply:
latest commit has these changes